tsconfig.json

```diff
@@ -20,6 +20,9 @@
     "experimentalDecorators": true,
     "jsx": "react",
     "baseUrl": ".",
+    "paths": {
+        "mattermost-redux/*": ["packages/mattermost-redux/src/*"],
```

Comment:
This rewrites the import paths for the type checker and for VS Code's TS plugin.